tutorials/create-cloud-build-image-factory-using-packer/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Create A Cloud Build Image Factory Using Packer
+description: Learn how to create an image factory using Google Cloud Build and Packer.
+author: johnlabarge
+tags: Cloud Build, Packer, Compute Engine, Image
+date_published: 2018-09-14
+---
+
+# Create A Cloud Build Image Factory Using Packer
+
+This tutorial will show you how to create an image factory using Cloud Build and [Packer by Hashicorp](https://packer.io). The image factory will automatically create new images from a Cloud Source Repository every time a new tag is pushed to that repository as depicted in the diagram below.
+
```

Comment:
I suggest you move the diagram up here before the prerequisites, so that there's less of a disconnect between the reference here and the diagram.